packages/compass-shell/src/components/shell-header/shell-header.jsx

```diff
@@ -8,9 +8,10 @@ import {
   keyframes,
   SpinLoader,
   withDarkMode,
+  useHotkeys,
```

Comment:
didn't know where to add this comment, but if we have a way to use the rendering without binding the keys, it would be nice to adapt also this file: https://github.com/mongodb-js/compass/blob/c7ee317492da4ebf944f7867a9bd8ca342a3c1ec/packages/compass-shell/src/components/shell-info-modal/keyboard-shortcuts-table.jsx#L31

Reply:
Yes, that will be great. Totally missed on this one.